eZ/Bundle/PlatformInstallerBundle/src/Resources/config_templates/clean/ezpublish.yml

```diff
@@ -21,6 +21,7 @@ ezpublish:
         # Available siteaccesses
         list:
             - site
+            - mobile
```

Comment:
Do we enable mobile by default now on the installed config for clean install? Looks like you only need it for tests, but this is the real config for install. Right?